msteams-platform/index-test.yml

```
@@ -0,0 +1,171 @@
+        - url: /learn/modules/embedded-web-experiences
+          itemType: learn
+          text: Tabs — create embedded web experiences 
+        - url:  /learn/modules/msteams-conversation-bots/
+          itemType: learn
+          text: Bots — create conversational interactions
+        - url: /learn/modules/msteams-messaging-extensions/
+          itemType: learn
+          text: Messaging extensions — create task-oriented interactions 
+        - url: /learn/modules/msteams-task-modules/
+          itemType: learn
+          text: Task modules — collect input and gather information
+        - url: /learn/modules/msteams-webhooks-connectors/
+          itemType: learn
+          text: Webhooks and Office 365 connectors — connect web services to channels and teams
+    # Card
+    - title: Develop
+      links:
+        - url: /microsoftteams/platform/tutorials/get-started-nodejs-app-studio
+          itemType: get-started
+          text: Build a Teams app using JavaScript/Node.js and App Studio
+        - url: /microsoftteams/platform/tutorials/get-started-dotnet-app-studio
+          itemType: get-started
+          text: Build a Teams app using C#/.NET and App Studio
+     # Card
+    - title: Deploy
+      links:
+        - url: /microsoftteams/platform/samples/app-templates
+          itemType: deploy
+          text: Teams app templates — production-ready applications
+        - url:  /microsoftteams/platform/samples/shifts-wfm-connectors
+          itemType: deploy
+          text: Teams shifts connectors - production-ready integrations
+        - url: /microsoftteams/platform/bots/how-to/add-power-virtual-agents-bot-to-teams
+          itemType: deploy
+          text: Integrate your Power Virtual Agents bot with Teams     
+    # Card
+    - title: Connect
+      links:
+        - url: https://developer.microsoft.com/microsoft-teams/blogs/
+          itemType: overview
+          text: Microsoft Teams developer blog
+        - url: https://aka.ms/microsoftteamscommunitycall
+          itemType: overview
+          text: Microsoft Teams Community Calls
+    # Card
+    - title: Community Help
+      links:
+        - url: /microsoftteams/platform/feedback
+          itemType: overview
+          text: Microsoft Teams Developer Community and response
+
+# tools section
+tools:
+  title: SDKs & Tools # < 60 chars (optional)
+  items:
+  # Card
+    - title: Microsoft Teams Toolkit extension for Visual Studio Code
+      # imageSrc should be square in ratio with no whitespace
+      imageSrc: /media/logos/logo_vs-code.svg
+      url: /microsoftteams/platform/toolkit/visual-studio-code-overview
+  # Card
+    - title: Microsoft Teams Toolkit extension for Visual Studio
+      # imageSrc should be square in ratio with no whitespace
+      imageSrc: /media/logos/logo_vs-ide.svg
+      url: /microsoftteams/platform/toolkit/visual-studio-overview
+    # Card
+    - title: JavaScript SDK for Tabs
```

```suggestion
    - title: JavaScript client SDK for tabs and other hosted experiences
```
Changed to align with title here: https://docs.microsoft.com/microsoftteams/platform/tabs/how-to/using-teams-client-sdk